apps/04TowersOfHanoi.js

```diff
@@ -91,6 +91,7 @@ function towersOfHanoi(startStack, endStack) {
       movePiece(startStack, endStack);
```

Comment:
having input other than the 'a' 'b' or 'c' would also warrant some checking you your function isValid. Great job with input scrubbing!